cmd/neofs-node/config.go

```diff
@@ -348,10 +343,10 @@ func initCfg(path string) *cfg {
 		tlsKeyFile  string
 	)
 
-	if viperCfg.GetBool(cfgTLSEnabled) {
+	if grpcconfig.TLS(appCfg).Enabled() {
 		tlsEnabled = true
-		tlsCertFile = viperCfg.GetString(cfgTLSCertFile)
-		tlsKeyFile = viperCfg.GetString(cfgTLSKeyFile)
+		tlsCertFile = grpcconfig.TLS(appCfg).CertificateFile()
```

Comment:
Lets calc `grpcconfig.TLS(appCfg)` once into var.

Reply:
moved in var section